packages/docs/package.json

```diff
@@ -19,11 +19,11 @@
     "precommit": "lint-staged --concurrent=false"
   },
   "dependencies": {
+    "@docsearch/js": "3",
```

Comment:
Can we use "^3.whatever.whatever"? No much difference, just for consistency.

Reply:
Hm, I'll check it

Reply:
Fixed